pkg/spec/util.go

```diff
@@ -38,3 +46,28 @@ func isPVCDefined(volumes []VolumeClaim, name string) bool {
 	}
 	return false
 }
+
+func GetScheme() (*runtime.Scheme, error) {
+	// Initializing the scheme with the core v1 api
+	scheme := api.Scheme
+	// Adding the batch scheme to support Jobs
+
+	// TODO: find a way where we don't have to add batch/v1 to the v1 scheme,
+	// instead we should be able to have different scheme for different controllers
+	if err := batch_v1.AddToScheme(scheme); err != nil {
+		return nil, errors.Wrap(err, "unable to add 'batch' to scheme")
+	}
+	return scheme, nil
+}
+
+func SetGVK(runtimeObject runtime.Object, scheme *runtime.Scheme) error {
```

Comment:
Can we have comment explaining what this function is for?

Reply:
added